ydb/docs/en/core/public-materials/videos.md

```diff
@@ -11,6 +11,9 @@ Video recordings from conferences and webinars. The materials are divided by cat
 
   {% include [FOSDEM](./_includes/conferences/2025/fosdem.md) %}
 
+  {% include [PGConfIndia](./_includes/conferences/2025/pgconfIndia.md) %}
```

Comment:
I'm not sure how consistent this is with the existing content, but let's change the order to be descending from newest to oldest?